styleguide/template.md

```diff
@@ -102,32 +102,34 @@ Use for inline code, language keywords, NuGet package names, command-line comman
 To link to a header in the same Markdown file (also known as anchor links), you'll need to find out the id of the header you're trying to link to. To confirm the ID, view the source of the rendered article, find the id of the header (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
 The id is auto-generated based on the header text. So, for example, given a unique section named `## Step 2`, the id would look like this `id="step-2"`.
 
-- Example: [Chapter 1](#chapter-1)
+- Example: `[Chapter 1](#chapter-1)` produces [Chapter 1](#chapter-1).
 
 To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
 
-- Example: [Readme file](../readme.md)
-- Example: [Welcome to .NET](../docs/welcome.md)
+- Example: `[Readme file](../readme.md)` produces [Readme file](../readme.md).
+- Example: `[Welcome to .NET](../docs/welcome.md)` produces [Welcome to .NET](../docs/welcome.md).
 
 To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
 
-- Example: [.NET Community](../docs/welcome.md#community)
+- Example: `[.NET Community](../docs/welcome.md#community)` produces [.NET Community](../docs/welcome.md#community).
```

Comment:
There is no community anchor. Perhaps open-source?

Reply:
Did you miss this comment, @BillWagner? It would be good for this anchor (as well as the anchor on line 105) to link to a real header in the document.  A good choice for the first, since it links to a multi-word header, is \#inline-code-blocks-with-language-identifier. For welcome.md, \#open-source.

Reply:
I did miss that. Just updated.